SUGGESTIONS.md

```diff
@@ -0,0 +1,17 @@
+# Suggestions
+
+### Project
+Great structure
+
+### README
+- Good work, needs more clearance on the structure
+
+### INSTALL SCRIPT
+- Missing - although not requested, could have saved me a time for a setup.
+
+### Helm
+
+- What's a point using helm `values.yaml` file if you are using static values in the charts ?
+- Why not use containers built in previous tasks from docker hub ?
+
```

Comment:
add a suppot more static value to values.yaml and add them to the relavent files.
change the deployment file to use the repostiroy container of details app instead of the local one